apps/nextjs/src/app/[locale]/boards/[name]/settings/_background.tsx

```diff
@@ -27,6 +31,12 @@ export const BackgroundSettingsContent = ({ board }: Props) => {
       backgroundImageSize: board.backgroundImageSize,
     },
   });
+  const medias = clientApi.media.getPaginated.useQuery({
+    page: 1,
+    pageSize: 100,
+    includeFromAllUsers: true,
+  });
```

Comment:
Doesn't this mean, that we only support 100 images, and the others won't be selectable?
I think we can easily reach 100, due to app icons being uploaded

Reply:
So you would use a bigger limit or what?

Reply:
Possibly, or implement server side filtering?

Reply:
Okay yes I'll implement this with the existing search functionality from the medias router, but then we probably also should show the name